src/components/modals/DistributeTokensModal.tsx

```diff
@@ -100,6 +100,7 @@ export default function DistributeTokensModal({
               mods={currentTicketMods}
               fundingCycle={currentFC}
               projectId={projectId}
+              reservedRate={metadata?.reservedRate ?? 0}
```

Comment:
**question**: Why do we need to divide by 2 in the above cases, but not for this case?

Reply:
@johnnyd-eth did you end up getting to the bottom of this?

Reply:
Oh my bad, this diff is out of date. Ignore me!